client/src/components/WeekListItem/WeekListItem.js

```diff
@@ -66,7 +66,7 @@ const WeekListItem = () => {
         />
       </Diamond>
       <DiamondLine />
-      <TextWeek href="/">Geschnetzeltes</TextWeek>
+      <TextWeek href="/Rezept">Geschnetzeltes</TextWeek>
     </ContainerWeek>
```

Comment:
Is this a link to a route? Then you should use Link to="/Rezept" instead of a href.  You need to import lik from react-router-dom. You can also style it with styled(Link)